airbyte-cdk/bulk/core/load/src/test/kotlin/io/airbyte/cdk/task/SpillToDiskTaskTest.kt

```diff
@@ -28,21 +27,19 @@ import kotlinx.coroutines.test.runTest
 import org.junit.jupiter.api.Assertions
 import org.junit.jupiter.api.Test
 
-@MicronautTest(environments = ["SpillToDiskTaskTest", "MockTempFileProvider"])
+@MicronautTest(
+    environments =
+        [
+            "SpillToDiskTaskTest",
+            "MockTempFileProvider",
+            "MockTaskLauncher",
```

Comment:
nonblocking: does the `class.simpleName` thing not work here?

Reply:
Unfortunately it doesn't. I thought it did because my intellij was running so slowly when I first checked it that the error didn't show up immediately.

We can either
A) just use names (drawback: name changes have to be manually maintained)
B) use constants (drawback: more stuff)

Reply:
mmk. let's go with strings for now - renaming is presumably not a thing we're doing very frequently in the long term, and I feel ok with "just be careful" as a short term strategy